links/views.py

```diff
@@ -18,5 +19,7 @@ def hello(request):
 
 def shortener_redirect(request, name):
     """Redirects to destination based on the name"""
-    destination = get_object_or_404(Link, name=name).destination
-    return redirect(destination)
+    link = get_object_or_404(Link, name=name)
+    if link.expiration_date and link.expiration_date < timezone.now():
```

Comment:
Let's use [the is_expired property](https://github.com/simonbru/ocurl-django/blob/15a7ffdb875a6736a3e1b14b801a2e61f5a8db11/links/models.py#L20)
```suggestion
    if link.is_expired:
```